### ISSUE TITLE
Grass clippings and impressions so far

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses-2#1]
Version 0.2 Beta

Thanks for the heads up last night...downloaded it this morning. Structures found so far..Root huts and small astral sorcery structures (the non fountain ones. Expanded my grass patch and use bonemeal to attempt to get grass clippings....wooden shears didn't work, although the grass disappeared. Gave myself iron shears and attempted it and still the same thing...grass disappeared...no clippings. Odd thing..no sound...like blocks breaking or clipping...just no sound. Has grass clipping been disabled? Oh thanks for the backpack being back!

Edit: In playing more with this, I found I get wildroot and terraspores but not one  grass clipping. Just odd that it doesn't have any sound.

**Comment by teagan75** [comment]
Sigh.....lol. No grass clipping hasn't been disabled. A weird glitch I came across while testing Roots is that nothing drops from breaking grass. I only disabled Pam's seeds and Vanilla seeds but nothing drops now. So, not sure if something is hard coded in the new HF to drop nothing. SO, I added the Scavenge mod but forgot to add you get the stuff only when you have an empty hand 🤦 I will fix.

I also had to fix a mortar recipe that wasn't giving the empty bucket back. No matter what script I write it won't allow it in the Roots mod. So, I changed the water bucket to Pam's Fresh Water. Fun Fun I will put out a hotfix today
